<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>levels</class>
 <widget class="QWidget" name="levels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>6</green>
        <blue>55</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>6</green>
        <blue>55</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>6</green>
        <blue>55</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>3</red>
        <green>6</green>
        <blue>55</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="level1">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>270</y>
     <width>662</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(93, 14, 65);
    border-style: solid;
    color:  rgb(255, 32, 78);;
    border-width: 5px;
    border-radius: 20px;
    border-color:  rgb(255, 32, 78);;
    font-size: 48px; /* Corrected from 'font: 48px' */
    padding: 6px; 
    background-repeat: no-repeat;
    background-position: center;
}
QPushButton::pressed {
    background-color: rgb(160, 21, 62); /* Background color when pressed */
}

</string>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level2">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>370</y>
     <width>662</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(93, 14, 65);
    border-style: solid;
    color:  rgb(255, 32, 78);;
    border-width: 5px;
    border-radius: 20px;
    border-color:  rgb(255, 32, 78);;
    font-size: 48px; /* Corrected from 'font: 48px' */
    padding: 6px; 
    background-repeat: no-repeat;
    background-position: center;
}
QPushButton::pressed {
    background-color: rgb(160, 21, 62); /* Background color when pressed */
}

</string>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level3">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>470</y>
     <width>662</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(93, 14, 65);
    border-style: solid;
    color:  rgb(255, 32, 78);;
    border-width: 5px;
    border-radius: 20px;
    border-color:  rgb(255, 32, 78);;
    font-size: 48px; /* Corrected from 'font: 48px' */
    padding: 6px; 
    background-repeat: no-repeat;
    background-position: center;
}
QPushButton::pressed {
    background-color: rgb(160, 21, 62); /* Background color when pressed */
}

</string>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level4">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>570</y>
     <width>662</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(93, 14, 65);
    border-style: solid;
    color:  rgb(255, 32, 78);;
    border-width: 5px;
    border-radius: 20px;
    border-color:  rgb(255, 32, 78);;
    font-size: 48px; /* Corrected from 'font: 48px' */
    padding: 6px; 
    background-repeat: no-repeat;
    background-position: center;
}
QPushButton::pressed {
    background-color: rgb(160, 21, 62); /* Background color when pressed */
}

</string>
   </property>
   <property name="text">
    <string>Level 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level5">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>670</y>
     <width>662</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(93, 14, 65);
    border-style: solid;
    color:  rgb(255, 32, 78);;
    border-width: 5px;
    border-radius: 20px;
    border-color:  rgb(255, 32, 78);;
    font-size: 48px; /* Corrected from 'font: 48px' */
    padding: 6px; 
    background-repeat: no-repeat;
    background-position: center;
}
QPushButton::pressed {
    background-color: rgb(160, 21, 62); /* Background color when pressed */
}

</string>
   </property>
   <property name="text">
    <string>Level 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>0</y>
     <width>1012</width>
     <height>311</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>32</green>
         <blue>78</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>32</green>
         <blue>78</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>64</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CHOOSE LEVEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>5</y>
     <width>1012</width>
     <height>311</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>76</red>
         <green>255</green>
         <blue>183</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>76</red>
         <green>255</green>
         <blue>183</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>64</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CHOOSE LEVEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(93, 14, 65);
    border-style: solid;
    color:  rgb(255, 32, 78);;
    border-width: 5px;
    border-radius: 20px;
    border-color:  rgb(255, 32, 78);;
    font-size: 48px; /* Corrected from 'font: 48px' */
    padding: 6px; 
    background-repeat: no-repeat;
    background-position: center;
}
QPushButton::pressed {
    background-color: rgb(160, 21, 62); /* Background color when pressed */
}

</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>title_2</zorder>
  <zorder>title_3</zorder>
  <zorder>level2</zorder>
  <zorder>level3</zorder>
  <zorder>level4</zorder>
  <zorder>level5</zorder>
  <zorder>back</zorder>
  <zorder>level1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
